### PR TITLE
fix(vpcgw):add better vizualisation for gateway and IP

### LINKED ISSUE #3009: Improve visualization of a vpc gateway
```
$ scw vpc-gw gateway get 28f30e4a-7fcb-428f-9d59-3cf4a0887736
ID                                       28f30e4a-7fcb-428f-9d59-3cf4a0887736
OrganizationID                           951df375-e094-4d26-97c1-ba548eeb9c42
ProjectID                                951df375-e094-4d26-97c1-ba548eeb9c42
CreatedAt                                2 weeks ago
UpdatedAt                                2 weeks ago
Type.Name                                VPC-GW-S
Type.Bandwidth                           100000000
Type.Zone                                fr-par-1
Status                                   running
Name                                     pgw-stoic-stonebraker
IP.ID                                    ba9f5d86-9d31-43b3-b86e-2253d4bb812c
IP.OrganizationID                        951df375-e094-4d26-97c1-ba548eeb9c42
IP.ProjectID                             951df375-e094-4d26-97c1-ba548eeb9c42
IP.CreatedAt                             2 weeks ago
IP.UpdatedAt                             2 weeks ago
IP.Address                               51.15.218.46
IP.Reverse                               46-218-15-51.instances.scw.cloud
IP.GatewayID                             28f30e4a-7fcb-428f-9d59-3cf4a0887736
IP.Zone                                  fr-par-1
GatewayNetworks.0.ID                     2a70ccda-7691-4041-82a8-99d82bbfda94
GatewayNetworks.0.CreatedAt              2 weeks ago
GatewayNetworks.0.UpdatedAt              19 minutes ago
GatewayNetworks.0.GatewayID              28f30e4a-7fcb-428f-9d59-3cf4a0887736
GatewayNetworks.0.PrivateNetworkID       2a0ab5e8-f2bc-4788-8f0e-a9601412fb5f
GatewayNetworks.0.MacAddress             02:00:00:12:75:b4
GatewayNetworks.0.EnableMasquerade       true
GatewayNetworks.0.Status                 ready
GatewayNetworks.0.DHCP.ID                1d3dc7bf-b4ce-4023-b625-7b80c5bd9300
GatewayNetworks.0.DHCP.OrganizationID    951df375-e094-4d26-97c1-ba548eeb9c42
GatewayNetworks.0.DHCP.ProjectID         951df375-e094-4d26-97c1-ba548eeb9c42
GatewayNetworks.0.DHCP.CreatedAt         2 weeks ago
GatewayNetworks.0.DHCP.UpdatedAt         2 weeks ago
GatewayNetworks.0.DHCP.Subnet            192.168.0.0/24
GatewayNetworks.0.DHCP.Address           192.168.0.1
GatewayNetworks.0.DHCP.PoolLow           192.168.0.2
GatewayNetworks.0.DHCP.PoolHigh          192.168.0.254
GatewayNetworks.0.DHCP.EnableDynamic     true
GatewayNetworks.0.DHCP.ValidLifetime     1 hours
GatewayNetworks.0.DHCP.RenewTimer        50 minutes
GatewayNetworks.0.DHCP.RebindTimer       51 minutes
GatewayNetworks.0.DHCP.PushDefaultRoute  false
GatewayNetworks.0.DHCP.PushDNSServer     false
GatewayNetworks.0.DHCP.DNSLocalName      priv
GatewayNetworks.0.DHCP.Zone              fr-par-1
GatewayNetworks.0.EnableDHCP             true
GatewayNetworks.0.Zone                   fr-par-1
GatewayNetworks.1.ID                     9dc6c8cd-274b-456e-acb9-cb4903dd94b0
GatewayNetworks.1.CreatedAt              2 weeks ago
GatewayNetworks.1.UpdatedAt              19 minutes ago
GatewayNetworks.1.GatewayID              28f30e4a-7fcb-428f-9d59-3cf4a0887736
GatewayNetworks.1.PrivateNetworkID       3ab3f34f-7159-4fff-b015-41d8c984f32d
GatewayNetworks.1.MacAddress             02:00:00:12:75:b5
GatewayNetworks.1.EnableMasquerade       true
GatewayNetworks.1.Status                 ready
GatewayNetworks.1.DHCP.ID                0dbb9474-7554-4bb1-b5ad-3f33e59cb51b
GatewayNetworks.1.DHCP.OrganizationID    951df375-e094-4d26-97c1-ba548eeb9c42
GatewayNetworks.1.DHCP.ProjectID         951df375-e094-4d26-97c1-ba548eeb9c42
GatewayNetworks.1.DHCP.CreatedAt         19 minutes ago
GatewayNetworks.1.DHCP.UpdatedAt         19 minutes ago
GatewayNetworks.1.DHCP.Subnet            192.168.1.0/24
GatewayNetworks.1.DHCP.Address           192.168.1.1
GatewayNetworks.1.DHCP.PoolLow           192.168.1.2
GatewayNetworks.1.DHCP.PoolHigh          192.168.1.254
GatewayNetworks.1.DHCP.EnableDynamic     true
GatewayNetworks.1.DHCP.ValidLifetime     1 hours
GatewayNetworks.1.DHCP.RenewTimer        50 minutes
GatewayNetworks.1.DHCP.RebindTimer       51 minutes
GatewayNetworks.1.DHCP.PushDefaultRoute  false
GatewayNetworks.1.DHCP.PushDNSServer     false
GatewayNetworks.1.DHCP.DNSLocalName      priv
GatewayNetworks.1.DHCP.Zone              fr-par-1
GatewayNetworks.1.EnableDHCP             true
GatewayNetworks.1.Zone                   fr-par-1
Version                                  0.2.24
BastionEnabled                           false
BastionPort                              61000
SMTPEnabled                              false
Zone                                     fr-par-1
```

It would be nice to have a nice visualization with section for each gateway

### PR DESCRIPTION
**Pull request by Laure-di**
close #3009

### CODE REVIEW

File: internal/namespaces/vpcgw/v1/custom_gateway_network.go [hunk 1]
```diff
@@ -17,3 +19,24 @@ var (
 		vpcgw.GatewayNetworkStatusUnknown:     &human.EnumMarshalSpec{Attribute: color.Faint},
 	}
 )
+
+func gatewayNetworkMarshallerFunc(i interface{}, opt *human.MarshalOpt) (string, error) {
+	type tmp vpcgw.Gateway
+	vpcgtwNetwork := tmp(i.(vpcgw.Gateway))
+	log.Println("get into the marshaller func")
```
Comment (yfodil) [inline internal/namespaces/vpcgw/v1/custom_gateway_network.go:26]:
```suggestion
```
Lost log message :p

File: internal/namespaces/vpcgw/v1/custom_gateway_network.go [hunk 2]
```diff
@@ -17,3 +17,22 @@ var (
 		vpcgw.GatewayNetworkStatusUnknown:     &human.EnumMarshalSpec{Attribute: color.Faint},
 	}
 )
+
+func gatewayNetworkMarshallerFunc(i interface{}, opt *human.MarshalOpt) (string, error) {
```
Comment (remyleone) [inline internal/namespaces/vpcgw/v1/custom_gateway_network.go:21]: Could you add a test with some goldens?

File: internal/testhelpers/helpers_vpcgw.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+)
+
+func CreateGateway(metakey string) core.BeforeFunc {
+	return func(ctx *core.BeforeFuncCtx) error {
+		res := ctx.ExecuteCmd([]string{"scw", "vpc-gw", "gateway", "create", "--wait"})
+		createGatewayResponse := res.(*vpcgw.Gateway)
+		ctx.Meta[metakey] = createGatewayResponse
+		return nil
+	}
+	//return core.ExecStoreBeforeCmd(
```
Comment (remyleone) [inline internal/testhelpers/helpers_vpcgw.go:15]: to remove

File: internal/namespaces/vpcgw/v1/testdata/test-vpc-gw-gateway-get-simple.golden [hunk 4]
```diff
@@ -0,0 +1,11 @@
+🎲🎲🎲 EXIT CODE: 1 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Cannot find resource 'gateway' with ID 'd7545d81-434d-40e4-9ab0-540aad823749'
+🟥🟥🟥 JSON STDERR 🟥🟥🟥
+{
```
Comment (remyleone) [inline internal/namespaces/vpcgw/v1/testdata/test-vpc-gw-gateway-get-simple.golden:5]: How can I test that the rendering is improved?

**Review by remyleone — APPROVED** [review]

**Pull request merged**